source/extensions/filters/common/expr/BUILD

```diff
@@ -16,7 +16,7 @@ envoy_cc_library(
         ":context_lib",
         "//source/common/http:utility_lib",
         "//source/common/protobuf",
-        "@com_google_cel_cpp//eval/public:builtin_func_registrar",
+        #"@com_google_cel_cpp//eval/public:builtin_func_registrar",
```

Comment:
why this change?

Reply:
never mind. was accidental. has been changed back.